bin/plot_truncated_iterations

```diff
@@ -0,0 +1,336 @@
+                    intersection = False
+                else:
+                    summary['steady_state_time'][last_iter]['same'] += 1
+                if intersection:
+                    summary['intersection'][last_iter]['same'] += 1
+                else:
+                    summary['intersection'][last_iter]['different'] += 1
+                continue
+            # One ALL NSS, one not.
+            if (all_nss(truncated_summaries[last_iter]['machines'][machine][vm][bench]['detailed_classification']) or
+                all_nss(truncated_summaries[DEFAULT_ITERS]['machines'][machine][vm][bench]['detailed_classification'])):
+                summary['steady_iter'][last_iter]['different'] += 1
+                summary['steady_state_time'][last_iter]['different'] += 1
+                summary['intersection'][last_iter]['different'] += 1
+                continue
+            # One ALL FLAT, one not.
+            if (all_flat(truncated_summaries[last_iter]['machines'][machine][vm][bench]['detailed_classification']) or
+                all_flat(truncated_summaries[DEFAULT_ITERS]['machines'][machine][vm][bench]['detailed_classification'])):
+                summary['steady_iter'][last_iter]['different'] += 1
+                summary['steady_state_time'][last_iter]['different'] += 1
+                summary['intersection'][last_iter]['different'] += 1
+                continue
+            # All bad inconsistent.
+            if (truncated_summaries[last_iter]['machines'][machine][vm][bench]['classification'] == 'bad inconsistent' and
+                truncated_summaries[DEFAULT_ITERS]['machines'][machine][vm][bench]['classification'] == 'bad inconsistent'):
+                if intersection:
+                    summary['intersection'][last_iter]['same'] += 1
+                else:
+                    summary['intersection'][last_iter]['different'] += 1
+                continue
+            # One bad inconsistent.
+            if (truncated_summaries[last_iter]['machines'][machine][vm][bench]['classification'] == 'bad inconsistent' or
+                truncated_summaries[DEFAULT_ITERS]['machines'][machine][vm][bench]['classification'] == 'bad inconsistent'):
+                summary['steady_iter'][last_iter]['different'] += 1
+                summary['steady_state_time'][last_iter]['different'] += 1
+                summary['intersection'][last_iter]['different'] += 1
+                continue
+            # All three measures should be available in both the DEFAULT_ITER and last_iter cases.
+            if do_iqrs_differ(truncated_summaries[DEFAULT_ITERS]['machines'][machine][vm][bench]['steady_state_iteration_iqr'],
+                              truncated_summaries[last_iter]['machines'][machine][vm][bench]['steady_state_iteration_iqr']):
+                summary['steady_iter'][last_iter]['different'] += 1
+                intersection = False
+            else:
+                summary['steady_iter'][last_iter]['same'] += 1
```

Comment:
No, at this stage we have gone through all the special cases and we need to test the CIs for the steady iter and time values.